packages/shorebird_cli/lib/src/commands/preview_command.dart

```diff
@@ -172,6 +180,15 @@ class PreviewCommand extends ShorebirdCommand {
     return app.appId;
   }
 
+  Future<String?> promptForFlavor(
+    List<String> flavors,
+  ) async {
+    return logger.chooseOne<String>(
+      'Which app flavor?',
+      choices: flavors,
+    );
+  }
```

Comment:
I'm not sure we gain much by wrapping this in its own function, as the call to `logger.chooseOne` is not substantially more complex than the call to `promptForFlavor`.